Sources/SwiftFormat/Rules/StandardizeDocumentationComments.swift

```diff
@@ -0,0 +1,239 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import Markdown
+import SwiftSyntax
+
+/// Reformats documentation comments to a standard structure.
+///
+/// Format: Documentation is reflowed in a standard format:
+/// - All documentation comments are rendered as `///`-prefixed.
+/// - Documentation comments are re-wrapped to the preferred line length.
+/// - The order of elements in a documentation comment is standard:
+///   - Abstract
+///   - Discussion w/ paragraphs, code samples, lists, etc.
+///   - Param docs (outlined if > 1)
+///   - Return docs
+///   - Throw docs
+@_spi(Rules)
+public final class StandardizeDocumentationComments: SyntaxFormatRule {
+  public override class var isOptIn: Bool { return true }
+
+  // For each kind of `DeclSyntax` node that we visit, if we modify the node we
+  // need to continue into that node's children, if any exist. These are
+  // different for different node types (e.g. an accessor has a `body`, while an
+  // actor has a `memberBlock`).
+
+  public override func visit(_ node: AccessorDeclSyntax) -> DeclSyntax {
```

Comment:
Does the Swift compiler or DocC process comments on accessor independently of those 
on the parent property/subscript?

Likewise for some of the other declarations included here, like `deinit` and `extension`.

I suppose it doesn't _hurt_ to include them here so we shouldn't necessarily exclude them, but I'm more curious if there are generally user expectations around these specific ones.

Reply:
Ah, not sure! I just grabbed the list of decls from... [here](https://github.com/swiftlang/swift-syntax/blob/f2a32a70802b9e96b24c62a8fb92525bcf12c8d4/Sources/SwiftSyntax/generated/SyntaxBaseNodes.swift#L156-L181). I'll try it out and see what happens – we could probably just include nominal declarations.